### PR TITLE
Fix --followings-output path

### PR DESCRIPTION
**Pull request by Roffild**
Fix absolute and relative paths for --followings-output

### CODE REVIEW

File: instagram_scraper/app.py [hunk 1]
```diff
@@ -1532,7 +1532,7 @@ def main():
     if args.followings_input:
         scraper.usernames = list(scraper.query_followings_gen(scraper.login_user))
         if args.followings_output:
-            with open(scraper.destination+scraper.followings_output, 'w') as file:
```
Comment (AlexNik) [inline instagram_scraper/app.py:1535]: open(os.path.join(scraper.destination, scraper.followings_output), 'w')

**Review by AlexNik — CHANGES_REQUESTED** [review]

File: instagram_scraper/app.py (hunk 1 above)
Comment (Roffild) [reply instagram_scraper/app.py:1535]: But it breaks the absolute path...

Comment (AlexNik) [reply instagram_scraper/app.py:1535]: Why?

Comment (Roffild) [reply instagram_scraper/app.py:1535]:
```python
os.path.join(r"C:\download", r"D:\abs\path") == r"D:\abs\path"
```
Well... OK...

**Review by AlexNik — APPROVED** [review]

**Pull request merged**